llama_index/indices/empty/base.py

```diff
@@ -48,6 +49,11 @@ def as_retriever(self, **kwargs: Any) -> BaseRetriever:
 
         return EmptyIndexRetriever(self)
 
+    def as_query_engine(self, **kwargs: Any) -> BaseQueryEngine:
+        if "response_mode" not in kwargs:
```

Comment:
we could probably enforce that the response_mode must equal generation, and raise ValueError if otherwise

Reply:
is emptyIndex going to be replaced by anything new or just removed.

It was nice to have this one actually, esp for testing

Reply:
added the ValueError